support.md

```diff
@@ -90,6 +90,7 @@ On Windows 95 or 95 Plus! you can enable `-O3` but not `-Ofast`. You'll also nee
 - Windows Home Server 2011
 - Windows Server 2012
 - Windows Server 2016
+- Windows 11. Only losers use windows 11
```

Comment:
Per [GitHub's community guidelines](https://docs.github.com/en/site-policy/github-terms/github-community-guidelines#maintaining-a-strong-community) (i.e if we don't want to get banned or something), it would be preferable, according to them, to "criticize ideas, not people". Rephrasing to emphasize the wrong of win11 instead of the wrong of it's user seems like a wise thing to do. *also gosh I need to stop using win11 someday...*